lxc/__init__.py

```diff
@@ -217,6 +217,9 @@ def create(self, template=None, flags=0, args=(), bdevtype=None):
             "args" (optional) is a tuple of arguments to pass to the
             template. It can also be provided as a dict.
         """
+        if(not self.defined):
```

Comment:
Should just be `if not self.defined:`

Reply:
I'd probably move it down to just before the `return _lxc.Container` line though so it's only called at the last moment.